Core/MIPS/JitCommon/JitCommon.cpp

```diff
@@ -157,10 +159,12 @@ const char *ppsspp_resolver(struct ud*,
 		*offset = addr - (uint64_t)MIPSComp::jit->GetBasePtr();
 		return "jitcode";
 	}
+#if defined(_M_IX86) || defined(_M_X64) || defined(MIPS)
```

Comment:
On what platform are you building that is not ARM, ARM64, MIPS, or x86/x64?  Just wondering.  Maybe it's pointless to define ppsspp_resolver on MIPS and non-Intel anyway.

-[Unknown]